velox/functions/prestosql/coverage/data/all_scalar_functions.txt

```diff
@@ -83,10 +88,14 @@ doy
 e
 element_at
 empty_approx_set
+ends_with
 enum_key
 exp
 expand_envelope
-features
```

Comment:
is 'features' no longer available?

```
> show functions like 'features';
 Function |    Return Type     |                                 Argument Types                                 | Function Type | Deterministic | Description | Variable Arity | Built In | Temporary | Language
----------+--------------------+--------------------------------------------------------------------------------+---------------+---------------+-------------+----------------+----------+-----------+----------
 features | map(bigint,double) | double                                                                         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double                                                                 | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double                                                         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double                                                 | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double                                         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double                                 | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double, double                         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double, double, double                 | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double, double, double, double         | scalar        | true          |             | false          | true     | false     |
 features | map(bigint,double) | double, double, double, double, double, double, double, double, double, double | scalar        | true          |             | false          | true     | false     |
(10 rows)
```

Reply:
I am getting this output currently:
```
presto:system> show functions like 'features';
 Function | Return Type | Argument Types | Function Type | Deterministic | Description | Variable Arity | Built In | Temporary | Language
----------+-------------+----------------+---------------+---------------+-------------+----------------+----------+-----------+----------
(0 rows)
```
Will recheck and update, thanks for sharing.